idc_index/index.py

```diff
@@ -177,6 +172,94 @@ def get_idc_version():
         idc_version = Version(idc_index_data.__version__).major
         return f"v{idc_version}"
 
+    @staticmethod
+    def _get_latest_idc_index_data_release_assets():
+        """
+        Retrieves a list of the latest idc-index-data release assets.
+
+        Returns:
+            release_assets (list): List of tuples (asset_name, asset_url).
+        """
+        release_assets = []
+        url = "https://api.github.com/repos/ImagingDataCommons/idc-index-data/releases/latest"
```

Comment:
I think here we should fetch the assets corresponding to the version of idc-index-data that is currently used by idc-index. I am not sure how to do this, but I think this is the right way to go.

Reply:
it can be accomplished very easily by 

f"https://api.github.com/repos/ImagingDataCommons/idc-index-data/releases/tags/{idc_index_data.__version__}"

Reply:
What I don't know is what version is going to be assigned to that variable for non-tagged releases. I think for non-tagged releases, it would be best to use assets from latest.

Reply:
I do not understand. Every release must have a tag.

Reply:
Yes, every release will have a tag, but it is also possible to install and use the package outside of a release, from a non-tagged git hash.

Reply:
Yes, but when they build from source (github), the pyproject toml still has locked idc-index-data version. https://github.com/ImagingDataCommons/idc-index/blob/main/pyproject.toml#L37

Reply:
I'd always prefer working with versioned data to non-versioned data. It helps debugging faster. Thats my 2 cents.

Reply:
Did what Vamsi suggested for now.